### PR TITLE
phraseapp 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by sacry-dyn**
New formula for Homebrew, created a new pull request for PhraseApp. See here for an older (rejected) pull request https://github.com/Homebrew/homebrew/pull/42269

### CODE REVIEW

**Comment by DomT4** [comment]
From the previous thread:

> A future submission should build from source using Go as well. We don't use upstream binaries, we produce our own.

:)

File: Library/Formula/phraseapp.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  url "https://github.com/phrase/phraseapp-client/releases/download/1.0.1/phraseapp_macosx_amd64.tar.gz"
+  version "1.0.1"
+  sha256 "4b1aa4c36f7c92618904d6882817e8459e3420f146e4864555dbe6ab9b65a192"
+
+  def install
+    bin.install "phraseapp"
+  end
+
+  test do
+    output = `#{bin}/phraseapp info`
```
Comment (DomT4) [inline Library/Formula/phraseapp.rb]: Should be able to use `shell_output` or `pipe_output` rather than backtics here.

**Comment by sacry-dyn** [comment]
Hi @DomT4 , 

thank you for the fast reply and feedback. We will find something out for the binary. The building of this client is a little bit more complicated because we generate code for several repositories and then make the binary (some of the repositories are private (the one that generates the code)) and others public. As for the test we will be on it. Does this PR stay open or do we need to make another pull request?

Regards

**Comment by DomT4** [comment]
Feel free to leave this one open whilst you work on it. If you  keep us semi-updated it won't be closed unless there's nothing we can do to get this into Homebrew or doing so becomes too convoluted.

**Comment by sacry-dyn** [comment]
Hey,

just to give you a small heads up. We are working on it, go 1.5 building gives us a small headache. We are looking into resolving this though.

Regards,
Matthias

**Comment by DomT4** [comment]
Thanks for the update!

**Comment by DomT4** [comment]
How's this going?

**Comment by sacry-dyn** [comment]
@DomT4 sorry for the late reply (holidays). We are still working on it.  This is a bit on "hold" as we need to evaluate our options on how to migrate to go 1.5, as it has still a few things we do not like about it and go 1.4 is more reasonable to use as of now. This will change soon but it can take a bit more time. We hope this is okay.

**Comment by DomT4** [comment]
No worries! As long as we're aware the PR isn't dead or likely to hang indefinitely it can stay open.

**Comment by bfontaine** [comment]
Anything new here?

**Comment by bfontaine** [comment]
Ping?

File: Library/Formula/phraseapp.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Phraseapp < Formula
+  desc "PhraseApp CLI Tool - API client binary"
+  homepage "https://phraseapp.com/cli"
+  url "https://github.com/phrase/phraseapp-client/releases/download/1.0.1/phraseapp_macosx_amd64.tar.gz"
```
Comment (bfontaine) [inline Library/Formula/phraseapp.rb]: We need to build from source.

**Comment by bfontaine** [comment]
Anything new? The latest release is now [1.1.8](https://github.com/phrase/phraseapp-client/releases/tag/1.1.8).

**Comment by mikemcquaid** [comment]
Let's get a newer release here and include it in a new PR.

**Pull request closed without merging**